### PR TITLE
Introducing unique selectors (#69) to production

### PR DESCRIPTION
**Pull request by dabrady**
We released #69 to Tier 1 on 17 June, and Tier 2 on 22 June; and things are looking good. This PR is to promote it to production to finalize the rollout.

We'll be rolling this out in 1 week, **on ~~29 June~~ 6 July,** if go according to plan.

### CODE REVIEW

File: test/testcafe/test-snippet-prod.js [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
+}('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
 
-    ProdPerfectKeen.ready(function() {
+ProdPerfectKeen.ready(() => {
+  const client = new ProdPerfectKeen({
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:6]: ignoring

File: test/testcafe/test-snippet-prod.js [hunk 2]
```diff
@@ -1,32 +1,28 @@
...
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
+}('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
 
-    ProdPerfectKeen.ready(function() {
+ProdPerfectKeen.ready(() => {
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:5]: ignoring

File: test/testcafe/test-snippet-prod.js [hunk 3]
```diff
@@ -1,32 +1,28 @@
...
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
+}('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:3]: ignoring

File: test/testcafe/test-snippet-prod.js [hunk 4]
```diff
@@ -1,32 +1,28 @@
...
-      var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:2]: ignoring

File: test/testcafe/test-snippet-prod.js [hunk 5]
```diff
@@ -1,32 +1,28 @@
-/* eslint-disable */
-    !function(name,path,ctx){
-      var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.appendChild(s)}}
-    }('ProdPerfectKeen','https://test.trackinglibrary.prodperfect.com/keen-tracking.min.js',this);
-/* eslint-disable */
+!function(name,path,ctx){
```
[the remark this answers is not in the document]
Comment (dabrady) [reply test/testcafe/test-snippet-prod.js:1]: ignoring

**Review by rhymiz — DISMISSED** [review]

**Review by rhymiz — DISMISSED** [review]

**Review by rhymiz — APPROVED** [review]

**Comment by dabrady** [comment]
I'm gonna close-and-reopen this PR to prompt a fresher Semaphore build, given we released a few bugfixes over the last two weeks.

**Comment by dabrady** [comment]
Hmm, well that didn't do anything. So I just clicked the "Rebuild last revision" button on the `canary-tier-2` Semaphore project:
https://semaphoreci.com/prodperfect/prodperfect-keen-tracking-js/branches/canary-tier-2/builds/24

**Comment by dabrady** [comment]
Huzzah, fourth time's the charm. Proceeding with the deploy!

**Comment by dabrady** [comment]
Production build is here: https://semaphoreci.com/prodperfect/prodperfect-keen-tracking-js/branches/production/builds/12

**Comment by dabrady** [comment]
Production deploy is here:
https://semaphoreci.com/prodperfect/prodperfect-keen-tracking-js/servers/manual-deploy-production-branch-to-production/deploys/6

**Pull request merged**